lib/KalturaModel.php

```diff
@@ -236,7 +236,18 @@ public function listPlayersUiConfs() {
 
 		$uiConfs = $this->_client->uiConf->listAction( $filter );
 
-		return $uiConfs;
+
+		// filter out playlist players
+		$players = new stdClass();
+		$players->objects = array();
+
+		foreach( $uiConfs->objects as $uiConf ) {
+			if( strpos($uiConf->tags, 'playlist') === false ) {
```

Comment:
I would parse the tags by first exploding it into an array (because strpos will also catch part of the tag). But this looks fine as well.